### DISCUSSION TITLE
Bossbar option to specialized commands

### DISCUSSION THREAD

**Discussion by bobhenl** [Zrips/CMI#4002] [Feature Requests]
**Description of feature/improvement request:**

---
### Detailed explanation of what it should do, when and in what situation: ###
Hi, I would like to see !bossbar in specialized commands. With options to set there color of bar as well. Would be nice if you can implement it.

### Basic explanation as to why this feature is needed and/or useful to have: ###
Would be useful as like normal specialized commands, but with option to use there bossbar too.

**Comment by CloudeLecaw** [comment]
Would be usefull but you can define also a secound command that posts a bossbarmsg

**Comment by bobhenl** [comment]
Yeah I did it by bossbarmsg, just suggesting that it could be added sometimes.
